### ISSUE TITLE
How can I reproduce same or similar mAP of yolov4-csp?

### ISSUE THREAD

**Issue by chjej202** [AlexeyAB/darknet#7511]
I tried to train yolov4-csp to get same/similar mAP which is shown in the README (width=512 height=512 64.8% mAP@0.5 (46.2% AP@0.5:0.95).

Because yolov4-csp shows higher FPS compared to yolov4 with the same image size and mAP is similar to yolov4, 
I expected to get similar mAP after training yolov4-csp, but I got  low mAP (55% mAP@0.5 with 512x512 image) than the mAP (64.8%) in the README file.

For this reason, I would like to know how to reproduce same/similar mAP shown in the README file.

I trained with [yolov4-csp.cfg](https://raw.githubusercontent.com/AlexeyAB/darknet/master/cfg/yolov4-csp.cfg) and I changed the learning rate from 0.001 to 0.00025 and burn_in from 1000 to 4000 after 10000 iterations to use 4 GPUs (2080 Ti) for training.
(Train set: COCO train 2017)

I can get a same mAP with pre-trained weights, but I would like to know how to train yolov4-csp to get same/similar mAP.
